### PR TITLE
add contributing.md and change license

### LINKED ISSUE #20: Consider using GNU General Public License v2.0 instead of MIT & describe the decision process in the documentation
Perhaps you can enhance and include the following "Ups and Downs" decision table:

MIT vs. GNU GPL in the context of fancy-flashcard
| Option                           | Ups        |  Downs |
| ------------------- | ------------------ | -------------- |
| MIT |  Simplest License    | Potential lack of motivation to publish improvements |
| GNU GPL |  Every party which adds improvements is obliged to publish those improvements. With this might come a quicker positive co-evolution.    | tbd |

### PR DESCRIPTION
**Pull request by FischerRene**
closes #20

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -1,21 +1,339 @@
...
+possible use to the public, the best way to achieve this is to make it
+free software which everyone can redistribute and change under these terms.
+
+  To do so, attach the following notices to the program.  It is safest
+to attach them to the start of each source file to most effectively
+convey the exclusion of warranty; and each file should have at least
+the "copyright" line and a pointer to where the full notice is found.
+
+    <one line to give the program's name and a brief idea of what it does.>
+    Copyright (C) <year>  <name of author>
```
Comment (nikolockenvitz) [inline LICENSE:294]: could you please update this and the below parts where some placeholder content is inserted?

Comment (FischerRene) [reply LICENSE:294]: Not sure if I don't understand the license correctly or you don't:
I think this is a placeholder for other users wanting to contribute, not for us to insert information into.
Please correct me if I am wrong.

**Comment by FischerRene** [comment]
was this last change what you had in mind?

File: LICENSE (hunk 1 above)
Comment (FischerRene) [reply LICENSE:294]: To prove my point, [take a look at linux GPL-2.0](https://github.com/torvalds/linux/blob/master/LICENSES/preferred/GPL-2.0).

Comment (nikolockenvitz) [reply LICENSE:294]: ok, seems valid, nvm

**Pull request merged**